packages/ui-kit/src/components/rss-feeds.js

```diff
@@ -1,23 +1,46 @@
 import React from 'react';
 import useSWR from 'swr';
-import Parser from 'rss-parser';
 import PropTypes from 'prop-types';
 import LoadingSpinner from '@commercetools-uikit/loading-spinner';
 import ContentNotifications from './content-notifications';
 import RssFeedTable from './rss-feed-table';
 
+// Inspired by https://css-tricks.com/how-to-fetch-and-parse-rss-feeds-in-javascript/
+async function parseRssFeed(url) {
+  const response = await fetch(url);
+  const rawBody = await response.text();
+  const data = new window.DOMParser().parseFromString(rawBody, 'text/xml');
+
+  const feedTitle = data.querySelector('title').firstChild.data;
```

Comment:
I went with this simpler approach `firstChild.data` (thanks for the suggestion) for accessing the text value.
We only have single children, so we don't need to use `childNodes`.